src/datatypes.ts

```diff
@@ -103,11 +113,10 @@ interface ILogger {
    * @return {ILogger}      the named logger
    */
   get(name: string): ILogger;
-  time(label: string): void;
-  timeEnd(label: string): void;
-  enabledFor(level: ILogLevel): boolean;
+
   createDefaultHandler(options?: ILoggerOpts): (messages: any[], context: IContext) => void;
+
+  setLevel(level: ILogLevel): void;
 }
 
-declare var Logger: ILogger;
-export = Logger;
+export type JSLoggerExportType = IJSLoggerDefaultsType & ILogger;
```

Comment:
Not a fan of this class name, why not just `JSLogger`?

Reply:
👍 library name makes sense here, I agree